.github/workflows/ibc-test.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
     timeout-minutes: 60
     env:
       SRC_DIR: ${{ github.workspace }}/ibc-test-src
-      AXON_COMMIT: d03d2bb7cb3dcdc03319c3a74beeee6715e7f448
+      AXON_COMMIT: 7ae0a826647d07e0b25b4adb440a8c97231d6a04
```

Comment:
Currently, the Axon version should contain this commit to [change the create contract limit config](https://github.com/axonweb3/axon/commit/56b1a218ea5ed0b84aae5e74422b4f77bcfc8b6e).

Would you try this Axon branch?
-> https://github.com/axonweb3/axon/commits/0.2.0-dev.1

Reply:
Related issue:
- https://github.com/synapseweb3/ibc-solidity-contract/issues/9